_test_artifacts/README.rst

```diff
@@ -0,0 +1,6 @@
+_test_artifacts directory
+========================
+
+this directory contains artifacts from tests
+
+any files dropped here will be uploaded in github action
```

Comment:
looks like I missed the change to ci.yml which makes this statement true, since these tests don't leave anything behind in _test_artifacts it is not needed, will make the ci.yml change in another branch where there are actual files left behind to archive